t/fuzz/fuzz_parseini.c

```diff
@@ -0,0 +1,37 @@
+#include "fio.h"
+
+static int initialized = 0;
+
+int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
```

Comment:
Can you move function braces on to the line below function (see fio.c as an example).

Reply:
ok